app/views/devise/sessions/new.html.erb

```diff
@@ -19,7 +21,7 @@
   <% end %>
 
   <div class="actions">
-    <%= f.submit "Log in", class:"btn btn-primary"  %>
+    <%= f.submit "Log in", class:"btn btn-primary", id:"submitbtn" %>
```

Comment:
Make habit to write/use the any customized name's with Camelcase, With underscore or with hyfen `submitBtn, submit_btn, submit-btn`